opendbc/car/rivian/interface.py

```diff
@@ -18,7 +18,7 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, experime
     CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
     ret.steerControlType = structs.CarParams.SteerControlType.torque
-    ret.radarUnavailable = True
+    ret.radarUnavailable = False
```

Comment:
fyi before merging, we'll want to keep this disabled by default until we start using it for longitudinal